examples/src/main/java/io/opencensus/examples/helloworld/QuickStart.java

```diff
@@ -0,0 +1,98 @@
+import io.opencensus.tags.Tagger;
+import io.opencensus.tags.Tags;
+import io.opencensus.trace.SpanBuilder;
+import io.opencensus.trace.Status;
+import io.opencensus.trace.Tracer;
+import io.opencensus.trace.Tracing;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Random;
+import java.util.logging.Logger;
+
+/** Simple program that collects data for video size. */
+public final class QuickStart {
+
+  private static final Logger logger = Logger.getLogger(QuickStart.class.getName());
+
+  private static final Tagger tagger = Tags.getTagger();
+  private static final ViewManager viewManager = Stats.getViewManager();
+  private static final StatsRecorder statsRecorder = Stats.getStatsRecorder();
+  private static final Tracer tracer = Tracing.getTracer();
+
+  // frontendKey allows us to break down the recorded data
+  private static final TagKey FRONTEND_KEY = TagKey.create("my.org/keys/frontend");
+
+  // videoSize will measure the size of processed videos.
+  private static final MeasureLong VIDEO_SIZE = MeasureLong.create(
+      "my.org/measure/video_size", "size of processed videos", "MBy");
+
+  // Create view to see the processed video size distribution broken down by frontend.
+  private static final View.Name VIDEO_SIZE_VIEW_NAME = View.Name.create("my.org/views/video_size");
+  private static final View VIDEO_SIZE_VIEW = View.create(
+      VIDEO_SIZE_VIEW_NAME,
+      "processed video size over time",
+      VIDEO_SIZE,
+      Aggregation.Distribution.create(
+          BucketBoundaries.create(
+              Arrays.asList(0.0, (double) (1 << 8), (double) (1 << 16)))),
+      Collections.singletonList(FRONTEND_KEY),
+      Cumulative.create());
+
+  /** Main launcher for the QuickStart example. */
+  public static void main(String[] args) {
+    TagContextBuilder tagContextBuilder =
+        tagger.currentBuilder().put(FRONTEND_KEY, TagValue.create("mobile-ios9.3.5"));
+    SpanBuilder spanBuilder = tracer.spanBuilder("my.org/ProcessVideo");
+    viewManager.registerView(VIDEO_SIZE_VIEW);
+
+    // Process video.
+    // Record the processed video size.
+    try (Scope scopedTags = tagContextBuilder.buildScoped();
+        Scope scopedSpan = spanBuilder.startScopedSpan()) {
+      tracer.getCurrentSpan().addAnnotation("Start processing video.");
+      // Sleep for [0,10] milliseconds to fake work.
+      Thread.sleep(new Random().nextInt(11));
+      statsRecorder.newMeasureMap().put(VIDEO_SIZE, 25648).record();
+      tracer.getCurrentSpan().addAnnotation("Finished processing video.");
+    } catch (Exception e) {
+      tracer.getCurrentSpan().addAnnotation("Exception thrown when processing video.");
+      tracer.getCurrentSpan().setStatus(Status.UNKNOWN);
+      logger.severe(e.getMessage());
+    }
+
+    ViewData viewData = viewManager.getView(VIDEO_SIZE_VIEW_NAME);
+    logger.info(
+        String.format("Recorded stats for %s:\n %s", VIDEO_SIZE_VIEW_NAME.asString(), viewData));
+  }
```

Comment:
Should we output the collected spans (e.g. using `LoggingTraceExproter`)? Or at least add a comment to pretend we are doing so? The caveats are that we need to specify a proper sampling policy, as well as `Thread.sleep()` for some time to wait for exportation. Both of these add to the complexity of the code.

Reply:
SG, I added a LoggingExporter to print spans to log. The only downside is we need to [wait for >5s](https://github.com/census-instrumentation/opencensus-java/blob/master/impl_core/src/main/java/io/opencensus/implcore/trace/export/ExportComponentImpl.java#L29) before spans are exporter, while in the Go example we only wait for [2s](https://github.com/census-instrumentation/opencensus-go/blob/master/examples/helloworld/main.go#L86).

Reply:
If you are not hurry to merge this change, I can create a PR to add a shutdownhook to export and flush the unexported data.

Reply:
That sounds great, I can hold off this PR till you make that change.

Reply:
Per our documentation meeting today, we would like to have this example ready ASAP. I'll leave a TODO to remove the 5.1s gap once we have the export shutdownhook.